tests/unit/checkpoint/test_lr_scheduler.py

```diff
@@ -53,7 +53,7 @@ def test_checkpoint_lr_scheduler(self, tmpdir, zero_stage, use_cpu_offload):
         }
         if get_accelerator().is_fp16_supported():
             config_dict["fp16"] = {"enabled": True}
-        elif get_accelerator().is_fp16_supported():
+        elif get_accelerator().is_bf16_supported():
```

Comment:
fyi @delock

Reply:
@delock ln54-56 was added by you in another PR. Why do we need it?

Reply:
@duli2012 the intention is to allow the test run on CPU which currently does not support FP16 datatype, but BF16 datatype is supported.  Thanks for fix the bug!